egs2/ms_indic_18/asr1/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+lang=te # te ta gu
+
+train_set=train_${lang}
+train_dev=dev_${lang}
+test_set="${train_dev} test_${lang}"
+
+asr_config=conf/train_asr.yaml
+lm_config=conf/train_lm.yaml
+inference_config=conf/decoder_asr.yaml
+
+if [[ "zh" == *"${lang}"* ]]; then
+  nbpe=2500
+elif [[ "fr" == *"${lang}"* ]]; then
+  nbpe=350
+elif [[ "es" == *"${lang}"* ]]; then
+  nbpe=235
+else
```

Comment:
You can remove these lines

Reply:
Make sense. I'll remove them. When I do additional experiments next week to fine-tune bpe, I can add them for individual languages